types/index.d.ts

```diff
@@ -2,7 +2,18 @@
  * Options for the getBinPath() function.
  */
 export interface BinaryOptions {
+  /**
+   * Name of the binary. Only needs to be specified when the package exports several binaries.
+   *
+   * Default: `package.json` `name` property
```

Comment:
Would it be possible to use JSDoc? Thanks :)

Reply:
Updated.